learn/tia.md

```diff
@@ -16,7 +16,7 @@ prev:
 | Total supply at genesis | 1,000,000,000 TIA                                                                                |
 | Inflation schedule      | 8% in the first year, decreasing 10% per year until reaching an inflation floor of 1.5% annually |
 | Decimals                | 6                                                                                                |
-| Conversion              | $\text{utia} = \text{TIA} \times 10^{-6}$                                                        |
+| Conversion              | $\text{1 utia} = \text{TIA} \times 10^{-6}$                                                        |
```

Comment:
```suggestion
| Conversion              | $\text{1 uTIA} = \text{TIA} \times 10^{-6}$                                                        |
```

Reply:
since the `BondDenom` is `utia` should we style it that way?

Reply:
I feel like no, since it's an internal programmer thing and this page is talking about the asset not in that context.